### ISSUE TITLE
save and upload arabert

### ISSUE THREAD

**Issue by NaglaaAbdelhady** [aub-mind/arabert#171]
i used arabert for emotion detection task, after i trained the model in large dataset i want to save this model and load it again to predict the label of unannotated text, how can i  accomplish this (save , load and predict)?
######################## Create the trainer
trainer = Trainer(
    model = model_init(),
    args = training_args,
    train_dataset = train_dataset,
    eval_dataset=test_dataset,
    compute_metrics=compute_metrics,
)

#start the training
trainer.train()
